src/styles/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(\images\star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n+1),
+.stars__star:nth-child(-n+2),
+.stars__star:nth-child(-n+3),
+.stars__star:nth-child(-n+4) {
+  background-image: url(\images\star-active.svg);
+}
```

Comment:
use nesting,

also still not resolved issue with active stars

modifier stars--{n} should have effect and change count of  active stars.

pass all styles from your `layout stars` task.

stars--4 (4 active)

stars--2(2 active) 
etc